spring-cloud-stream-samples/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<module>tap</module>
 		<module>double</module>
 		<module>extended</module>
+		<module>rxjava</module>
```

Comment:
I am not sure if we want to have this `sample` module into SCSM repo. Even in XD case, we only have the moving average sample in XD [samples repo] (https://github.com/spring-projects/spring-xd-samples)